clients/js/evm.ts

```diff
@@ -74,19 +64,9 @@ export async function query_contract_evm(
       result.isInitialized = await tb.isInitialized(result.implementation)
       result.tokenImplementation = await tb.tokenImplementation()
       result.chainId = await tb.chainId()
-      // TODO: need new sdk release to expose this function in BridgeImplementation
-      const tb2 = new ethers.Contract(contract_address, ["function finality() public view returns (uint8)", "function evmChainId() public view returns (uint256)", "function isFork() public view returns (bool)"], provider)
-      result.finality = await tb2.finality()
-      try {
-        result.evmChainId = await tb2.evmChainId()
-      } catch (e) {
-        result.evmChainId = null
-      }
-      try {
-        result.isFork = await tb2.isFork()
-      } catch (e) {
-        result.isFork = null
-      }
+      result.finality = await tb.finality()
+      result.evmChainId = (await tb.evmChainId()).toString()
```

Comment:
`evmChainId()` returns a `BigNumber` and `.toString()` returns the decimal representation